framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Commands/GenerateProxyCommand.cs

```diff
@@ -56,6 +57,12 @@ public async Task ExecuteAsync(CommandLineArgs commandLineArgs)
 
             var uiFramework = GetUiFramework(commandLineArgs);
 
+            output = commandLineArgs.Options.GetOrNull(Options.Output.Short, Options.Output.Long);
+            if (!string.IsNullOrWhiteSpace(output) && (!output.EndsWith("/") || !output.EndsWith("\\")))
```

Comment:
Instead of this and using `output + ""`, why not use [Path.Combine](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=netcore-3.1)?